config.template/faf-coturn/faf-coturn.env

```diff
@@ -1,2 +1,6 @@
-EXTERNAL_IP=127.0.0.1
-SKIP_AUTO_IP=true
+TURN_PORT=3478
+TURN_PORT_START=10000
+TURN_PORT_END=20000
+TURN_SECRET=banana
+TURN_SERVER_NAME=faf-coturn
+TURN_REALM=faforever.com
```

Comment:
The template should always be usable from localhost. Is this realm somehow checked or used to connect to?

Reply:
the realm is the realm in which the user account has to exist, if the user doesn't specify a realm, the server will reply with an error and its realm so the user can retry using this realm, we don't use realms, so it really doesn't matter what is specified there, but it needs to be specified to use lt-cred-mech